mobile/components/views/Attractions/tools/openingHourMapper.js

```diff
@@ -0,0 +1,16 @@
+const mapArrayToJson = (openingHoursArray) => {
+    const transformedHours = {};
+    
+    openingHoursArray.forEach((dayData) => {
+      const { day, openingHour, closingHour } = dayData;
+      const dayKey = day.toLowerCase(); // Convert day to lowercase for consistency
+      transformedHours[dayKey] = {
+        openingHour,
+        closingHour,
+      };
+    });
+  
+    return transformedHours;
+  };
+
+export default mapArrayToJson;
```

Comment:
If its normal function, its better to do `export const` instead of `export default` at the end.